### PR TITLE
[WIP]User roles

### PR DESCRIPTION
**Pull request by jadekstewart3**
### Describe your change in plain English.
Edit database to reflect new architecture. Re work models and fixtures to work with new db. Tests are not currently passing. 

### Link to the issue
#72

### CODE REVIEW

**Comment by Thrillberg** [comment]
@jadekstewart3 Let me know if you want help on this PR! I can pair sometime if that might help.

**Comment by jadekstewart3** [comment]
@Thrillberg I would LOVE that! I'm available all day tomorrow!

**Comment by edwinthinks** [comment]
Closing to rethink about how we can do this in the future <3
